db/seeds/run-seed.js

```diff
@@ -1,9 +1,10 @@
 const devData = require('../data/development-data/index.js');
+const testData = require('../data/test-data')
```

Comment:
Please put this file back to how it was. You have a before each in your test file that correctly seeds the test DB with the test data. This run seed file is designed to seed the dev DB so it should use the dev data. 😄

Reply:
Ah right yes makes sense! Thanks